### PR TITLE
Add support for pod tolerations

### PR DESCRIPTION
**Pull request by idlewis**
First part of support for https://github.com/WASdev/websphere-liberty-operator/issues/652

### CODE REVIEW

File: api/v1/runtimecomponent_types.go [hunk 1]
```diff
@@ -153,6 +153,10 @@ type RuntimeComponentSpec struct {
 	// Disable information about services being injected into the application pod's environment variables. Default to false.
 	// +operator-sdk:csv:customresourcedefinitions:order=27,type=spec,displayName="Disable Service Links",xDescriptors="urn:alm:descriptor:com.tectonic.ui:booleanSwitch"
 	DisableServiceLinks *bool `json:"disableServiceLinks,omitempty"`
+
+	// Tolerations to be added to application pods.
```
Comment (leochr) [inline api/v1/runtimecomponent_types.go:157]: Suggestion: Tolerations to be added to the application pods. Tolerations allow the scheduler to schedule pods on nodes with matching taints.

**Review by leochr — COMMENTED** [review]
@idlewis Looks good. Just one minor suggestion for the description. Thank you

**Comment by idlewis** [comment]
Test PR: https://github.ibm.com/websphere/operators/pull/158

**Review by leochr — APPROVED** [review]
@idlewis Thank you. Looks good.

**Pull request merged**